CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+We mostly adhere to [Keep a Changelog 1.1.0](https://keepachangelog.com/en/1.0.0/)
+but use the following change types: "Added", "Changed", "Fixed" & "Dependencies Updated".
+
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased](https://github.com/orca-services/cakephp-swagger-ui/)
```

Comment:
@Richard-Strittmatter don't you think it would be better to link to the list of commits in the main branch?

https://github.com/orca-services/cakephp-swagger-ui/commits/main